engines/drascula/interface.cpp

```diff
@@ -94,6 +127,11 @@ bool DrasculaEngine::confirmExit() {
 	centerText(_textsys[1], 160, 87);
 	updateScreen();
 
+	Common::TextToSpeechManager *ttsMan = g_system->getTextToSpeechManager();
+	if (ConfMan.getBool("tts_enabled") && ttsMan != nullptr) {
+		ttsMan->say(_textsys[1]);
```

Comment:
It would be better replace `1` with something like `VERB_LOOK` or similarly named constant